ext/socket/raddrinfo.c

```diff
@@ -366,7 +366,11 @@ rb_getaddrinfo_a(const char *node, const char *service,
 	if (ret) return ret;
 
 	arg.req = &req;
-	arg.timeout = timeout;
+    if(timeout != NULL) {
+        arg.timeout = timeout;
+    } else {
+        arg.timeout = NULL;
+    }
```

Comment:
This change seems no meanings.

Reply:
Thanks for pointing it out, you're right.